test/11_array.jl

```diff
@@ -650,4 +650,19 @@ end
     @test unique(x) == collect(1:10)
 end
 
+### WIP
```

Comment:
Put this in 13_arraycommon.jl. Why is it WIP ?

Reply:
it is not - except its location

Reply:
done